docs/developer-docs/latest/setup-deployment-guides/file-structure.md

```diff
@@ -40,10 +40,10 @@ The default structure of a Strapi project created without the starter CLI looks
 ├──── <a href="/developer-docs/latest/development/admin-customization.html#build">build</a> <span class="token comment"># build of the admin panel</span>
 ├──── config <span class="token comment"># API configurations</span>
 │     ├──── src
-│     │     ├ <a href="/developer-docs/latest/setup-deployment-guides/configurations/optional/cronjobs.html">cron-tasks.js</a>
 │     │     └ response-handlers.js
```

Comment:
Wondering if we should not simply remove the `src` folder completely, as response-handlers.js is not documented.

Reply:
Sure I can do that :)